testsuite/tests/typing-layouts/layout_poly.ml

```diff
@@ -721,19 +718,15 @@ Error: "[@layout_poly]" on this external declaration has no
 external[@layout_poly] makearray_dynamic : ('a : any). int -> 'a -> 'a array =
   "%makearray_dynamic"
 [%%expect{|
-Lines 1-2, characters 0-22:
-1 | external[@layout_poly] makearray_dynamic : ('a : any). int -> 'a -> 'a array =
-2 |   "%makearray_dynamic"
-Error: This construct requires the alpha version of the extension "layouts", which is disabled and cannot be used
+external makearray_dynamic : ('a : any). int -> 'a -> 'a array
```

Comment:
Could use a test that `%makearray_dynamic_uninit` requires alpha

Reply:
I'm not quite sure what to do here now that we've moved these to beta - @ccasin can you confirm?  (See lines 715 onwards of `typing-layouts/layout_poly.ml`)

Reply:
I've pushed a commit that updates this test appropriately.